README.md

```diff
@@ -11,41 +11,71 @@ using [Syft](https://github.com/anchore/syft).
 
 By default, this action will execute a Syft scan in the workspace directory
 and upload a workflow artifact SBOM in SPDX format. It will also detect
-if being run during a `release` and attach the SBOM
-as a release asset.
+if being run during a [GitHub release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)
+and upload the SBOM as a release asset.
 
 ## Example Usage
 
-### Scan a docker image
+### Scan a container image
 
-Use the `image` parameter
+To scan a container image using the docker daemon use the `image` parameter:
 
 ```yaml
 - uses: anchore/sbom-action@main
   with:
     image: example/image_name
 ```
 
+To use an specific container registry, prefix the image with the domain:
+
+```yaml
+- uses: anchore/sbom-action@main
+  with:
+    image: ghcr.io/example/image_name:tag
+```
+
+To authenticate with the registry, add credentials using
+`registry-username` and `registry-password`:
+
+```yaml
+- uses: anchore/sbom-action@main
+  with:
+    image: my-registry.com/my/image
+    registry-username: mr_awesome
+    registry-password: ${{ secrets.REGISTRY_PASSWORD }}
+```
+
+To connect directly to a registry without using the docker daemon,
+prefix the `image` name with `http://` or `https://`:
```

Comment:
Question: I'm not used to seeing image references that include `http[s]` — is this a convention you're seeing that we should follow?

Reply:
I haven't really seen it anywhere, but it is literally the URL to pull from, no? Still open to suggestions here, but to allow use of `registry:` there must be some way to configure it, an option `registry: direct | no-tls | http`?